302-quarkus-vertx-jwt/pom.xml

```diff
@@ -15,6 +15,40 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-auth-jwt</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-auth-common</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-vertx-web</artifactId>
```

Comment:
This dependency brings two others "quarkus-vertx-..." as transitive. Does it make to leave only this one and remove others?